### ISSUE TITLE
I am trying to write samples of audio recorded by wasapi loopback capture by different wave format

### ISSUE THREAD

**Issue by mithunvs** [filoe/cscore#363]

**Comment by mithunvs** [comment]
Please help

**Comment by JohnsonGao** [comment]
Yes, you can , there is an example in Samples named as RecordWithSepcificFormat ,  demoed to flexible format settings. But you know , WASAPI Loopback Recording only support the format alreay internal use, so if you really need to set format different to the system internal using , you have to convert it by yourself.
